serialization.go

```diff
@@ -71,13 +75,43 @@ func transformOut[T any](ctx context.Context, ans T) (T, error) {
 	return ans, nil
 }
 
-// Send sends a string response.
-func Send(w http.ResponseWriter, text string) {
-	_, _ = w.Write([]byte(text))
+// Send sends a response.
+// The format is determined by the Accept header.
+func Send(w http.ResponseWriter, r *http.Request, ans any) error {
+	switch parseAcceptHeader(r.Header.Get("Accept"), ans) {
+	case "application/xml":
+		return SendXML(w, r, ans)
+	case "text/html":
+		return SendHTML(r.Context(), w, ans)
+	case "text/plain":
+		return SendText(w, ans)
+	case "application/json":
+		SendJSON(w, ans)
+	case "application/yaml":
```

Comment:
We'd want `case "application/x-yaml", "text/yaml; charset=utf-8", "application/yaml": // https://www.rfc-editor.org/rfc/rfc9512.html` here as well right?

Reply:
Good point, I'm adding it!